pkg/client/cmd/deploy.go

```diff
@@ -338,3 +349,34 @@ func deployList(cmd *cobra.Command, args []string) {
 	}
 	table.Render()
 }
+
+func deployRollback(cmd *cobra.Command, args []string) {
+	if len(args) != 1 {
+		cmd.Usage()
+		return
+	}
+	appName := args[0]
+
+	revision, err := cmd.Flags().GetString("revision")
+	if err != nil || revision == "" {
+		client.PrintErrorAndExit("invalid revision parameter")
+	}
+
+	conn, err := connection.New(cfgFile, cfgCluster)
+	if err != nil {
+		client.PrintErrorAndExit("Error connecting to server: %v", err)
+	}
+	defer conn.Close()
+
+	req := &dpb.RollbackRequest{
+		AppName:  appName,
+		Revision: revision,
+	}
+	cli := dpb.NewDeployClient(conn)
+	_, err = cli.Rollback(context.Background(), req)
+	if err != nil {
+		client.PrintErrorAndExit(client.GetErrorMsg(err))
+	}
+
+	fmt.Println("rollback made with success")
+}
```

Comment:
Maybe `rollback done`, what do you think @arnaldopereira ?

Reply:
Looks better :)